docstrings_to_openapi/flask_routes.py

```diff
@@ -0,0 +1,102 @@
+import builtins
+from collections import defaultdict
+import functools
+import os
+from pprint import pprint
+import sys
+
+
+def _clean_up_methods(methods):
+    autoadded_methods = frozenset(['OPTIONS', 'HEAD'])
+    if methods <= autoadded_methods:
+        return methods
+    return methods.difference(autoadded_methods)
+
+
+def _get_routes(app):
+    endpoints = []
+
+    for rule in app.url_map.iter_rules():
+        url = str(next(app.url_map.iter_rules(rule.endpoint)))
+
+        # Exclude static endpoint
+        static_url_path = app.static_url_path
+        if rule.endpoint == 'static' and url.startswith(static_url_path):
+            continue
+
+        url_with_endpoint = (url, rule.endpoint)
+        if url_with_endpoint not in endpoints:
+            endpoints.append(url_with_endpoint)
+
+    endpoints.sort()
+
+    unique_view_fn_names = [e for _, e in endpoints]
+
+    for view_fn_name in unique_view_fn_names:
+        methodrules = defaultdict(list)
+
+        for rule in app.url_map.iter_rules(view_fn_name):
+            path = rule.rule
+            for method in _clean_up_methods(rule.methods):
+                methodrules[method].append(path)
+
+        for method, paths in methodrules.items():
+            view_fn = app.view_functions[view_fn_name]
+            view_docstring = view_fn.__doc__
+            if view_docstring:
+                yield method, paths, view_fn_name, view_docstring
+
+
+def _import_object(import_name):
```

Comment:
I would discourage against starting function names with underscores. What does it mean, are these functions somehow more special? This applies to the entire PR.

Reply:
It's a 'handshake' practice in most python code bases if you want to give a hint that it's a private function and not to be used outside the file